scripts/operations.py

```diff
@@ -66,7 +66,7 @@ def sample(job):
 
 def auto(job):
     "This is a meta-operation to execute multiple operations."
-    from my_project import get_project
+    from my_project.project import get_project
```

Comment:
I'm surprised that this is causing an issue since the get_project() function is exposed in the package's root name space, but it might also be related to absolute import in python2.7.